src/BiotSavartBCs.jl

```diff
@@ -47,7 +47,7 @@ mutable struct BiotSimulation <: AbstractSimulation
     end
 end
 import WaterLily: sim_step! # new type dispatch
-function sim_step!(sim::BiotSimulation;remeasure=true)
+function sim_step!(sim::BiotSimulation;remeasure=true,udf=nothing,kwargs...)
```

Comment:
so how are you passing the convective scheme `λ` into `biot_mom_step!`? I do not see it here?